views/policies.ecr

```diff
@@ -7,113 +7,120 @@
-                federation-upstream-set.</span>
-            </a> |
-            <a class="arg-tooltip" data-tag="federation-upstream-set">Federation upstream set
-              <span class="tooltiptext">Chooses the name of a set of upstreams to use with federation, or "all" to use
-                all
-                upstreams. Incompatible with federation-upstream.</span>
-            </a> |
-            <a class="arg-tooltip" data-tag="delivery-limit" data-value="0">Delivery limit
-              <span class="tooltiptext">The number of times a message can be redelivered before dropped or
-                dead-lettered</span>
-            </a> |
-            <a class="arg-tooltip" data-tag="max-age" data-value="1M">Max age
-              <span class="tooltiptext">Stream queue segments will be deleted when all messages in the segmented is older than this.
-                Valid units are Y(ear), M(onth), D(ays), h(ours), m(inutes), s(seconds).
-                Segments are only deleted when new messages are published to the stream queue.</span>
-            </a>
+        <section class="card regular-card">
+          <div class="table-wrapper">
+            <div id="table-error"></div>
+            <table id="table" class="table">
+              <thead>
+                <tr>
+                  <th>Virtual host</th>
+                  <th data-sort-key="name" class="left">Name</th>
+                  <th class="left">Pattern</th>
+                  <th class="left">Apply to</th>
+                  <th class="left">Definition</th>
+                  <th class="left">Priority</th>
+                  <th></th>
+                </tr>
+              </thead>
+              <tbody></tbody>
+            </table>
           </div>
-        </label>
-        <label>
-          <span>Priority</span>
-          <input type="number" name="priority" value="0" max="10" min="-10" step="1" required>
-        </label>
-        <button type="submit" class="btn-primary when-create">Add policy</button>
-        <div class="when-edit">
-          <button type="submit" class="btn-primary">Update</button>
-          <button type="reset" class="btn-danger cancel">Cancel</button>
-        </div>
-      </form>
-    </main>
-    <% render "partials/footer" %>
+        </section>
+        <form method="put" id="createPolicy" class="form card regular-card">
+          <h3 class="title-m">
+            <span class="when-create">Add policy</span>
+            <span class="when-edit">Edit policy</span>
+          </h3>
+          <div class="py-4">
+            <label class="when-create mb-4">
+              <span class="field-label">Virtual host</span>
+              <select class="dropdown" id="addPolicyVhost" name="vhost" required></select>
+            </label>
+            <label class="mb-4">
+              <span class="field-label">Name</span>
+              <input class="regular-text" type="text" name="name" required data-primary-key="">
```

Comment:
To me this looks way worse than before, I'm afraid it will make non-experts in our CSS styles hesitant to do any changes to this. I would prefer that we use the minimal amount of classes and use the fact that we are in a form with a certain class, then we know how we want to style the label element, the input element etc. Why do we need to repeat the `class="field-label"` every time for example? What does `mb-4` mean?